source/assets/Components/Home.js

```diff
@@ -202,38 +278,91 @@ function getEmail(username, token, userEmail) {
  * @param {String} username
  * @param {String} token
  */
-function deleteUser(username, token) {
-  let msg = {
-    type: 'deleteUser',
-    username: username,
-    token: token,
-  };
-
-  /**
-   * Redirects to the user login page after deleting user
-   */
-  function afterDelete() {
-    window.location.href = 'userLogin.html';
-    //setFormMessage(loginForm, 'error', 'Invalid username or password!');
-  }
-
-  POST(msg, afterDelete);
-}
+// function deleteUser(username, token) {
+//   let msg = {
+//     type: 'deleteUser',
+//     username: username,
+//     token: token,
+//   };
+
+//   /**
+//    * Redirects to the user login page after deleting user
+//    */
+//   function afterDelete() {
+//     window.location.href = 'userLogin.html';
+//     //setFormMessage(loginForm, 'error', 'Invalid username or password!');
+//   }
+
```

Comment:
I have added it in the edit user profile page, I'm happy leaving it on the home page if we want but the function wasn't being used and a lint check was not passing because of that so I just commented it out.

Reply:
I removed it